telegram/_business.py

```diff
@@ -183,3 +185,210 @@ def de_json(cls, data: Optional[JSONDict], bot: "Bot") -> Optional["BusinessMess
         data["chat"] = Chat.de_json(data.get("chat"), bot)
 
         return super().de_json(data=data, bot=bot)
+
+
+class BusinessIntro(TelegramObject):
+    """
+    This object represents the intro of a business account.
```

Comment:
the api for some reason has no description about the class, but I think it's good that we have something.